### DISCUSSION TITLE
Adding Biosemi as an input device

### DISCUSSION THREAD

**Discussion by chat2iandaly** [umair-hassan/best-toolbox#45] [Q&A]
Hi all,

I'm trying to add a biosemi EEG amplifier as an input device to the BEST toolbox but I'm not sure about the correct configuration. Is the correct solution to assume the connection is made via TCP/IP and enter the port number displayed in the biosemi software? I don't have a BOSS device and so I guess I will need to make use of the FieldTrip real-time buffer device type right?

Many thanks,
Ian

**Comment by umair-hassan** [comment]
Hi Ian,

Youll need to configure fieldtrip real-time buffer with BioSemi amps and configure FieldTrip buffer as hardware input device
